design/process/review-guidelines.md

```diff
@@ -8,40 +8,55 @@ editor: markdown
 dateCreated: 2021-05-28T21:02:49.613Z
 ---
 
-The design team posts new specs to [GitHub Discussions](https://github.com/centerofci/mathesar/discussions/categories/review-request) when there are new designs to review. All [core team](/team) members are encouraged to review new designs. This page describes the process.
+The design team creates specs to describe the design solutions related to the Mathesar product. Whether small or large, all design issues should be accompanied by a spec containing the solution details and information relevant to its implementation. Every spec goes through a review process so that the team can discuss it and ask questions.
 
-# Review Process
+The spec creation process begins when a design issue is started and finalizes when the spec is approved and merged. When a design issue is ready for review, a spec is created, and a PR is opened. Finalizing a design spec means, in most cases, that the design issue has also been solved. If this is not the case, the reason and additional requirements must be communicated in the design issue and the spec.
```

Comment:
Please add something about needing to create additional GitHub issues to track unresolved requirements in case the design issue is not solved.

Reply:
It might also be worth explaining what a "design issue" is; new contributors may not be familiar with our process of creating GitHub issues with the `work: design` label to track design problems.